repose-aggregator/core/repose-core/src/test/java/org/openrepose/nodeservice/request/RequestHeaderServiceContextTest.java

```diff
@@ -55,8 +52,6 @@
 import static org.mockito.Matchers.eq;
 import static org.mockito.Mockito.*;
 
-@RunWith(PowerMockRunner.class)
-@PowerMockIgnore("javax.management.*")
```

Comment:
Please add that to any scala tests you see without it. if it's not there it's not running in the gradle build